.azure/pipelines/components-e2e-tests.yml

```
@@ -44,9 +44,9 @@ jobs:
       displayName: NPM install
     - script: .dotnet/dotnet build ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-restore
       displayName: Build
-    - script: .dotnet/dotnet test ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-build --filter QuarantinedTest\!~github --logger trx
+    - script: .dotnet/dotnet test ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-build --filter Quarantined\!=true --logger trx
```

Missed an update, sorry
```suggestion
    - script: .dotnet/dotnet test ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-build --filter 'Quarantined!=true|Quarantined=false' --logger trx
```

---

Note: I'm not positive this is necessary. It's also unclear why we are using different options in eng\QuarantinedTests.BeforeArcade.props and eng\helix\content\RunTests\TestRunner.cs. @HaoK❔

---

Given this is linux/macos, I believe the `\` is required for `!`.

---

Pipeline is now passing, going to merge as is without this additional change, let me know if you disagree.

https://dev.azure.com/dnceng/public/_build/results?buildId=1489638&view=logs&j=fe94c0c9-bb8c-5d6f-3b51-887173cc2f5c&t=b4bd3469-f776-5941-b623-568498d38bde

---

Results look _almost_ right. Unfortunately, the `Quarantined=true` run put it's results in the same directory as this one. Probably need to add something more to that command line to make it possible for the second test results upload to work and to make the Tests tab easier to read.

---

(Root cause of problems here is use of `dotnet test` on build agents. We use Arcade infrastructure, including `$(ArtifactsTestResultsDir)` support everywhere else.)

---

> Given this is linux, I believe the `\` is required for `!`.

nit: You ignored the single quotes in my suggestion.